libraries/GCS_MAVLink/GCS.h

```diff
@@ -187,6 +188,9 @@ class GCS_MAVLINK
 {
 public:
     friend class GCS;
+#if AP_DDS_VISUALODOM_ENABLED
```

Comment:
We really should work out whether to call it external or visual.  I've already said I'd prefer external :-)

Reply:
Yea, but I'd prefer to keep consistent for now, and handle a mass-rename of visual to external in another PR. That ok?

Reply:
Talked to @tridge , we would like to keep it the same for now, and handle this after GSoC.